ray-operator/config/manager/manager.yaml

```
@@ -53,8 +53,10 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 100Mi
+            # One user said that a cluster with ~50 worker Pods (16 cores and
+            # 64 GB memory each) requires 500 MB memory.
```

```suggestion
            # Anecdotally, managing 500 Ray pods requires roughly 500MB memory.
            # Monitor memory usage and adjust as needed.
```

We wouldn't expect the size of the pods to matter.

---

The guy said 10 clusters of 50 pods each.

---

Wow. I forget the number of clusters.